docs/api-conventions.md

```diff
@@ -84,27 +109,39 @@ ports:
 
 
-Special Resources
------------------
+### Simple
+
+Every simple kind returned by the server, and any simple kind sent to the server that must support idempotency or optimistic concurency, SHOULD have the following metadata in a nested object field called "metadata":
+
+* resourceVersion: a string that identifies the common version of the objects returned by in a list. This value MUST be treated as opaque by clients and passed unmodified back to the server. A resource version is only valid within a single namespace on a single kind of resource.
 
-Kubernetes MAY return two resources from any API endpoint in special circumstances. Clients SHOULD handle these types of objects when appropriate.
+Since simple resources are often used as input alternate actions that modify objects, the resource version of the simple resource should correspond to the resource version of the object.
 
-A "Status" object SHOULD be returned by an API when an operation is not successful (when the server would return a non 2xx HTTP status code). The status object contains fields for humans and machine consumers of the API to determine failures. The information in the status object supplements, but does not override, the HTTP status code's meaning.
+
+Returning Errors
+----------------
+
+Kubernetes MAY return the status kind from any API endpoint. Clients SHOULD handle these types of objects when appropriate.
+
+A "Status" kind SHOULD be returned by an API when an operation is not successful (when the server would return a non 2xx HTTP status code). The status object contains fields for humans and machine consumers of the API to determine failures. The information in the status object supplements, but does not override, the HTTP status code's meaning.
 
 TODO: More details (refer to another doc for details)
 
 
-Synthetic Resources
--------------------
+Differing Representations
+-------------------------
+
+An API may represent a single entity in different ways for different clients, or transform an object after certain transitions in the system occur. In these cases, one request object may have two representations available as different resources, or different kinds. An example is a Pod, which represents the intent of the user to run a container with certain parameters. When Kubernetes schedules the Pod, it creates a Binding (a simple kind) that ties that Pod to a single host in the system. After this occurs, the pod is represented by two distinct resources - under the original Pod resource the user created, as well as in a BoundPods object that the host may query but not update. Also, a "pod status" resource may accept a PUT with the "pod" kind, with different rules about what fields may be changed.
```

Comment:
Binding is going to be a sub-resource under the pod, so it's similar to the status example, but I'm ok with that.